plugins/callback/yaml.py

```diff
@@ -19,6 +19,13 @@
       - default_callback
     requirements:
       - set as stdout in configuration
+    seealso:
+      - plugin: ansible.builtin.default
+        plugin_type: callback
+        description: >
+          There is a parameter C(result_format) in P(ansible.builtin.default#callback) that allows you to change the output format to YAML.
+    notes:
+      - With ansible-core 2.13 or newer, you can instead specify C(yaml) for the parameter C(result_format) in P(ansible.builtin.default#callback).
```

Comment:
```suggestion
      - With ansible-core 2.13 or newer, you can instead specify V(yaml) for the parameter O(ansible.builtin.default#callback:result_format) in P(ansible.builtin.default#callback).
```

Reply:
Oh, we can indicate the parameters directly. There are a lot of things I don't know. Thanks!